### PR TITLE
add controlwhitespaces.py

### PR DESCRIPTION
**Pull request by illuz**
Control white spaces in a string.

### CODE REVIEW

File: controlwhitespaces.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python3
+"""control the whitespaces in sing"""
```
Comment (brennerm) [inline controlwhitespaces.py]: change sing to string

File: controlwhitespaces.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+s = 'The Little Price'
+
+# justify stirng to be at least width wide
```
Comment (brennerm) [inline controlwhitespaces.py]: change stirng to string

File: controlwhitespaces.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# by adding whitespace
+width = 20
+s1 = s.ljust(width)
+s2 = s.rjust(width)
+s3 = s.center(width)
+print(s1)   # 'The Little Price    '
+print(s2)   # '    The Little Price'
+print(s3)   # '  The Little Price  '
+
+# strip whitespace in two sides of sing
```
Comment (brennerm) [inline controlwhitespaces.py]: change sing to string

**Comment by brennerm** [comment]
pls have a look at my notes

**Comment by illuz** [comment]
Sorry for that. I fixed it.

**Comment by brennerm** [comment]
Thanks :)

**Pull request merged**